extensions/amp-analytics/0.1/test/test-visibility-impl.js

```diff
@@ -638,6 +638,66 @@ describe('amp-analytics.visibility', () => {
       });
     });
 
+    it('"hidden" trigger should work with duration condition', () => {
```

Comment:
Sorry I meant without any conditions at all. so something like this:

```
          "hidden": {
            "on": "hidden",
            "visibilitySpec": {
              "selector": "#img-2"
            },
          }
```

A test like that might exist but was not sure and hence brought it up.